bundle/manifests/operator.openshift.io_certmanagers.yaml

```diff
@@ -34,6 +34,263 @@ spec:
           spec:
             description: CertManagerSpec defines the desired state of CertManager
             properties:
+              cainjectorConfig:
+                description: "CAInjectorConfig specifies further customization options
+                  for the cainjector's deployment spec. \n Possible customizations
+                  include the following, \n For OverrideArgs, --namespace string                          If
+                  set, this limits the scope of cainjector to a single namespace.
+                  If set, cainjector will not update resources with certificates outside
+                  of the configured namespace. --v Level                                   number
+                  for the log level verbosity"
+                properties:
+                  overrideArgs:
+                    items:
+                      type: string
+                    type: array
+                  overrideEnv:
+                    items:
+                      description: EnvVar represents an environment variable present
+                        in a Container.
+                      properties:
+                        name:
+                          description: Name of the environment variable. Must be a
+                            C_IDENTIFIER.
```

Comment:
Is is possible to have regular expression pattern?

Reply:
Yes, kubebuilder does provide regex pattern. `+kubebuilder:validation:Pattern:
string` . ([ref](https://book.kubebuilder.io/reference/markers/crd-validation.html))